cider/featurizer.py

```diff
@@ -594,49 +599,84 @@ def feature_plots(self, read_from_disk: bool = False) -> None:
         Args:
             read_from_disk: whether to load features from disk
         """
+        
+        def check_feature_existence(keys, names, feature_dataframe):
+            existing_keys = []
+            existing_names = []
+            for key, name in zip(keys, names):
+                if key in feature_dataframe.columns:
+                    existing_keys.append(key)
+                    existing_names.append(name)
+                    
+            return existing_keys, existing_names
+
         if read_from_disk:
             self.load_features()
 
         # Plot of distributions of CDR features
         if self.features['cdr'] is not None:
-            features = ['cdr_active_days__allweek__day__callandtext', 'cdr_call_duration__allweek__allday__call__mean',
-                        'cdr_number_of_antennas__allweek__allday']
+            
+            # Features are given different names by spark and bandicoot featurization logic; here we check
+            # which exists.
```

Comment:
feels like the right move here is to standardize the feature names across spark and bandicoot rather than this type of patching here